src/schemata/definitions/workflow-template/action-objects/update-resource.yml

```diff
@@ -21,6 +22,24 @@ additionalProperties:
             - workflow_run.data.id
             - deviceField.id
     - oneOf:
+        - type: object
+          required:
+            - type
+            - attributes
+          properties:
+            type:
+              type: string
+              enum: [device]
```

Comment:
I understand the the current customer case is for devices, but should we allow this for other resources with `name` also? @ignatiusreza

Reply:
Looking through the list, there doesn't seem to be an obvious need to update most or all other resources in the context of a workflow run. Jared confirmed that he sees no need for that, so let's keep it to devices for now.